src/metabase/handler.clj

```diff
@@ -22,7 +24,11 @@
   "The primary entry point to the Ring HTTP server."
   ;; ▼▼▼ POST-PROCESSING ▼▼▼ happens from TOP-TO-BOTTOM
   (->
-   #'routes/routes                         ; the #' is to allow tests to redefine endpoints
+   ;; when running TESTS use the var so we can redefine routes as needed. No need to waste time with repetitive var
+   ;; lookups when running normally
+   (if config/is-test?
+     #'routes/routes
+     routes/routes)
```

Comment:
Performance improvement: we had been doing var resolution of `routes/routes` on every API request to make it easier for tests to hot-swap the var. That's silly. Only use the var to enable hot-swapping when running tests. 

This shaves off a few fractions of a millisecond for each request which isn't much but it isn't nothing either